src/EnsembleKalmanInversion.jl

```diff
@@ -118,7 +118,8 @@ end
 """
     update_ensemble!(
         ekp::EnsembleKalmanProcess{FT, IT, Inversion},
-        g::AbstractMatrix{FT};
+        g::AbstractMatrix{FT},
+        process_type::Inversion;
```

Comment:
Note this is not the `process_type` it is the `process`

You have chosen to dispatch using an _object of type ::Inversion_ rather than just the _type ::Inversion_

Nothing wrong with this but we should change the variable name to `process`